### ISSUE TITLE
Authentication just spins

### ISSUE THREAD

**Issue by tjreo** [jeena/Bungloo#42]
Here are the sequence of events as best as I can remember them. Most of these tasks happened earlier in the weekend. Step five (5) happened today.
1. Installed Tentia for the first time
2. Entered "tjreo.tent.is" as my Tent Address. (I did NOT include the protocol)
3. The spinner on the logon screen spun forever.
4. Closed and reopened the app a number of times.
5. Tried adding the protocol: "https://tjreo.tent.is". No change in behavior. Just spinning.
6. Read a post from ^jeena.net an hour or so ago saying that the login button doesn't work and to use enter instead. Tried using enter but no change in behavior. Just spinning.

[Here is a gist](git://gist.github.com/4060255.git) containing all Tentia error messages from console.app.

**Comment by tjreo** [comment]
This is really, really weird. I had closed Tentia an hour ago. I just opened Tentia. Safari automatically opened to the tent.is authentication page. 

I authenticated Tentia and it is now working. 

I am a software developer and trainer by trade. I know how to develop and test. I have no explanation as to how this happened. All I can say is the First Run Experience needs some work :)

Now on to the fun stuff...

**Comment by jeena** [comment]
Ok thank you for your detailed report. Sadly the link to the gist doesn't work. I think I already fixed the stuff you are talking about yesterday actually when I added some more tests while the user authenticates.
- It now has a look if you put in either http:// or https:// if not you get an error telling you that
- It checks if this particular entity has a Link to your /profile, if not it tells you that

The thing why Safari opened automatically is fixed to that behaviour was due the fact that at startup the app loads the entity into the TextView and doing that triggers the onChange event which then automatically triggers the automatic login if you haven't been logged in. Due to the [Login]-button bug it saved your with https:// updated entity but before didn't trigger the login, so this was a combination of several bugs, I'm sorry for that and I hope the changes I did will prevent this to happen to others.
